docs/source/guide.rst

```diff
@@ -0,0 +1,54 @@
+User Guide
+==========
+
+This guide is a starting point for configuring and running the `test-operator`.
+
+Installation
+------------
+
+.. code-block:: bash
+
+    make install
+
+Execution
+---------
+Execute the following command to run the operator. Note, that after running the
+following command you will need to switch to another terminal unless you run it
+in the background.
+
+.. code-block:: bash
+
+    make run
+
+.. note::
+    You can run this step with the installation one at once by: ``make install run``
```

Comment:
Maybe something like this: "You can run this step together with the installation at once by running: " (?)